config/initializers/octobox_config.rb

```diff
@@ -0,0 +1,12 @@
+module Octobox
```

Comment:
Does this need to load before any other configs?

If so, might be helpful to prefix this with a `02`